bootstrap/templates/new/mix.exs

```diff
@@ -20,7 +20,8 @@ defmodule <%= app_module %>.Mixfile do
      config_path: "../../config/config.exs",
      lockfile: "../../mix.lock",<% else %>
```

Comment:
You know, I thought about this. And I was curious if this was the case, or actually a contribution to why I'm really wasn't working. I guess you're right, but in the sense of umbrellas there is a lot of assumption of what can be run at the top of it, where targets would blow other projects minds